test/cpp/jit/test_save_load.cpp

```diff
@@ -263,6 +263,15 @@ TEST(SerializationTest, ParentDirNotExist) {
       "Parent directory ./doesnotexist does not exist.");
 }
 
+TEST(SerializationTest, SaveLoadTest) {
+  std::stringstream ss;
+  torch::Tensor to_save = torch::ones(5);
+  torch::save(to_save, ss);
+  torch::Tensor to_load;
+  torch::load(to_load, ss);
+  ASSERT_TRUE(to_save.equal(to_load));
+}
```

Comment:
Hmm, how this test (that serializes tensors into the string stream) helps you test that file is serializable by the full path? Also, wasn't the issue about Python side of things rather than C++ one

Reply:
Changed the test to trying to save under an unlikely to exist driver folder. And the reported problem was under libtorch, so the C++ side.